packages/runner/src/lib/SecRunner.ts

```diff
@@ -0,0 +1,97 @@
+import { SecScanOptions } from './SecScanOptions';
+import { SecScan } from './SecScan';
+import {
+  HttpCommandDispatcher,
+  HttpCommandDispatcherConfig
+} from '@secbox/bus';
+import {
+  CommandDispatcher,
+  Configuration,
+  ConfigurationOptions
+} from '@secbox/core';
+import { Repeater, RepeaterFactory, RepeatersManager } from '@secbox/repeater';
+import { Reporter, StdReporter } from '@secbox/reporter';
+import { ScanFactory } from '@secbox/scan';
+
+export class SecRunner {
+  private readonly configuration: Configuration;
+  private repeater: Repeater | undefined;
+  private repeaterFactory: RepeaterFactory | undefined;
+  private repeatersManager: RepeatersManager | undefined;
+
+  get repeaterId(): string | undefined {
+    return this.repeater?.repeaterId;
+  }
+
+  constructor(config: Configuration | ConfigurationOptions) {
+    this.configuration =
+      config instanceof Configuration ? config : new Configuration(config);
+  }
+
+  public async init(): Promise<void> {
+    if (this.repeatersManager && this.repeaterFactory) {
+      throw new Error('Already initialized.');
+    }
+
+    await this.initConfiguration(this.configuration);
+
+    this.repeatersManager =
+      this.configuration.container.resolve(RepeatersManager);
+    this.repeaterFactory =
+      this.configuration.container.resolve(RepeaterFactory);
+
+    this.repeater = await this.repeaterFactory.createRepeater();
+    await this.repeater.start();
+  }
+
+  public async clear(): Promise<void> {
+    try {
+      if (this.repeater && this.repeatersManager) {
+        await this.repeater.stop();
+        await this.repeatersManager.deleteRepeater(this.repeater.repeaterId);
+      }
+    } finally {
+      this.repeater = undefined;
+      this.repeatersManager = undefined;
+      this.repeaterFactory = undefined;
```

Comment:
Let's use the `delete` function instead
```suggestion
      delete this.repeater;
      delete this.repeatersManager;
      delete this.repeaterFactory;
```